catalog-info.yaml

```diff
@@ -99,6 +99,46 @@ spec:
         publish_commit_status: false
         trigger_mode: none
 ---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-elasticsearch-periodic-java-ea
```

Comment:
Why have this as a separate pipeline instead of just adding it to the existing JVM matrix testing pipeline?